yarn.lock

```diff
@@ -2420,16 +2424,17 @@
     yup "^0.29.3"
 
 "@backstage/core@^0.3.0":
-  version "0.4.4"
+  version "0.3.2"
```

Comment:
Hold up these `yarn.lock` changes aren't right 😅 
Should revert to the ones from master and then run `yarn install`